draft-ietf-httpbis-http2bis.xml

```diff
@@ -417,15 +421,14 @@
         <figure anchor="FrameLayout">
           <name>Frame Layout</name>
           <artwork type="inline"><![CDATA[
- +-----------------------------------------------+
- |                 Length (24)                   |
- +---------------+---------------+---------------+
- |   Type (8)    |   Flags (8)   |
- +-+-------------+---------------+-------------------------------+
- |R|                 Stream Identifier (31)                      |
- +=+=============================================================+
- |                   Frame Payload (0...)                      ...
- +---------------------------------------------------------------+
+HTTP Frame {
+  Length (24),
+  Type (8),
+  Flags (8),
+  R (1),
```

Comment:
Do we need to call this `R` in the new format? Can we use our extra horizontal space and write out the whole word "Reserved"`?

Reply:
Works for me, as long as it doesn't rock anybody's world.